apps/ai/README.md

```
@@ -172,20 +174,28 @@ POST /v1/llm/analysis-results/describe
 - 라우터: `routers/analysis_summary.py`
 - OpenAPI 스펙: `/docs` 에서 200/422/502/500 응답 모두 노출됨
 
-### 채워야 할 함수 — `summarize_analysis_result`
+### LLM 구현체
 
-`services/analysis_summary.py`
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)